contract/io/Cargo.toml

```
@@ -0,0 +1,11 @@
+[package]
+name = "channel-io"
+version = "0.1.0"
+edition = "2021"
+license = "GPL-3.0"
```

We tend to provide examples under the MIT license.
```suggestion
license = "MIT"
```